### DISCUSSION TITLE
Migration command fails after saleor 3.0 update

### DISCUSSION THREAD

**Discussion by RGuven** [saleor/saleor#9627] [Q&A] [answered]
I'm working on Ubuntu 20.04 and giving this error -> 
``` django.db.utils.OperationalError: could not translate host name "db" to address: Temporary failure in name resolution  ```

![image](https://user-images.githubusercontent.com/37475770/165260895-46b1ef9a-bfac-4e9a-b92b-6796496d231d.png)

What can I do ?

**Comment by karolkielecki** [comment]
Hi @RGuven, it looks like an issue with Docker. Perhaps you can find a solution here: https://stackoverflow.com/questions/68548725/docker-django-db-utils-operationalerror-could-not-translate-host-name-db-to-a

**Comment by RGuven** [comment, marked as answer]
[SOLVED]

Everything is working properly right now. I stopped all images and then restarted. I used these commands.
  
  * docker kill $(docker ps -q) # stop all containers
  * docker rm $(docker ps -a -q) # remove all containers 
  * docker rmi $(docker images -q) # remove all images
  * docker network prune # remove all networks
  * docker volume prune # remove all volumes
